app2/src/main/java/com/example/recipebox/MainActivity.kt

```diff
@@ -21,23 +19,31 @@ class MainActivity : ComponentActivity() {
                 Surface(
                     modifier = Modifier.fillMaxSize(),
                     color = MaterialTheme.colors.background
+
                 ) {
-                    Greeting("Android")
+                    Scaffold()
                 }
             }
+
         }
     }
 }
 
 @Composable
-fun Greeting(name: String) {
-    Text(text = "Hello $name!")
+fun Scaffold() {
```

Comment:
I would recommend removing the default IntelliJ greeting function and using the scaffold method so you can create the type of layout you need as depicted in the design document.